proposals/sample-proposal/README.md

```diff
@@ -1,5 +1,36 @@
+| Title | Author | Date |
+| --- | --- | --- |
+| Title of proposal | `Your name` | today |
+
 # PROPOSAL NAME
 
-## Links:
-Discussion: [Discourse Forum](https://forum.theodinproject.com/c/sprints)
-Other Links
+## Summary
+
+Brief explanation of the proposal.
+
+## Motivation
+
+Explain the motivation behind this proposal. What is the purpose of this change or implementation? What does it solve? What is the expected outcome of this proposal?
+
+Make sure to express the motivation of this proposal in the best way you can so that if the change or implementation that is suggested here is not accepted, this motivation can be used to come up with an alternative solution.
+
+## Implementation
```

Comment:
I'm a little bit hesitant about going into details about the implementation in the proposals. Just because I personally think they are best left to when the team is formed around a proposal  and they get together to break down how they are going to approach.

I definitely could be wrong about that though and I'm more than happy to give it ago. What do the rest of the @TheOdinProject/odinmods think?

Reply:
I think it's better to lean towards less prescriptive implementation details. The team handling the proposal should decide on the approach themselves.

Reply:
That is what the "champions" are for IMO, this could just be a rough idea of steps to completion